core/osm_connector.py

```diff
@@ -314,7 +338,7 @@ def _build_stop(self, stop, stop_type):
 
         # Make sure name is not empty
         if 'name' not in stop.tags:
-            stop.tags['name'] = Stop.NO_NAME
+            stop.tags['name'] = "[" + self.stop_no_name + "]"
```

Comment:
Why use the square brackets here? When you don't search for stop names and there's stops without name, there will be square brackets in the name. Is that intentional?